packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -137,14 +126,14 @@ export class FunctionInfoGatherer implements ParametersGatherer<FunctionInfo> {
     return {
       type: 'CONTINUE',
       data: {
-        fileName: name,
         language
       }
     };
   }
 }
 
 const parameterGatherer = new CompositeParametersGatherer(
+  new SelectFileName({ prompt: nls.localize('force_function_enter_function') }),
```

Comment:
This command is more restrictive and only allows lowercase letters and numbers. Will need to change to be able to account for that.